tests/test-extensions/heisenberg-extension/src/main/java/org/mule/test/heisenberg/extension/HeisenbergOperations.java

```diff
@@ -266,6 +266,10 @@ public void ignoredOperation() {
 
   }
 
+  public void throwError() {
+    throw new AssertionError();
```

Comment:
It should be LinkageError, I was testing and tried both. The AssertionError was not previously handled but it is from this change on.